pkg/daemon/node.go

```diff
@@ -49,19 +50,32 @@ func getNodeAnnotation(client corev1.NodeInterface, node string, k string) (stri
 	return getNodeAnnotationExt(client, node, k, false)
 }
 
-// getNodeAnnotationExt is like getNodeAnnotation, but allows one to customize ENOENT handling
-func getNodeAnnotationExt(client corev1.NodeInterface, node string, k string, allowNoent bool) (string, error) {
+// GetNode gets the node object.
+func GetNode(client corev1.NodeInterface, node string) (*core_v1.Node, error) {
```

Comment:
you can lowercase the func name here as it's only used within `pkg/daemon`

Reply:
Hmm, true. Though WDYT about getting this in, and we just work on https://github.com/openshift/machine-config-operator/pull/409#issuecomment-463372473 as a follow up?

Reply:
That's ok as well (though I'm missing context on how to use informers for these scenarios) /me goes to learn